Sources/SourceKitLSP/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ target_sources(SourceKitLSP PRIVATE
   Swift/CodeActions/AddDocumentation.swift
   Swift/CodeActions/ConvertIntegerLiteral.swift
   Swift/CodeActions/ConvertJSONToCodableStruct.swift
+  Swift/CodeActions/ConvertStringConcatenationToStringInterpolation.swift,
```

Comment:
I think this the Windows build is failing because this shouldn’t have a comma.

Reply:
My bad, I've pushed a new commit, please test again.